### PR TITLE
PA - WhiteDay2010

### PR DESCRIPTION
**Pull request by Jump-Suit**
Adds PremiumAgency support for WhiteDay2010!

### CODE REVIEW

File: BackendProject/WebAPIs/PREMIUMAGENCY/Custom.cs [hunk 1]
```diff
@@ -10,12 +10,22 @@ public class Custom
         public static string? setUserEventCustomPOST(byte[]? PostData, string? ContentType, string workpath, string eventId)
         {
             string? output = null;
+
+            LoggerAccessor.LogInfo($"setUserEventCustom POSTDATA: \n{PostData}");
+
```
Comment (GitHubProUser67) [inline BackendProject/WebAPIs/PREMIUMAGENCY/Custom.cs:15]: This must be removed, can cause app instability is postdata is very large.

File: BackendProject/WebAPIs/PREMIUMAGENCY/Custom.cs [hunk 2]
```diff
@@ -10,12 +10,22 @@ public class Custom
...
             switch (eventId)
             {
                 case "95":
                     if (File.Exists($"{workpath}/eventController/MikuLiveEvent/setUserEventCustom.xml"))
                         output = File.ReadAllText($"{workpath}/eventController/MikuLiveEvent/setUserEventCustom.xml");
-                    LoggerAccessor.LogError($"[PREMIUMAGENCY] - SetUserEventCustom sent for MikuLiveEvent {eventId}!");
+                    LoggerAccessor.LogInfo($"[PREMIUMAGENCY] - SetUserEventCustom sent for MikuLiveEvent {eventId}!");
+
```
Comment (GitHubProUser67) [inline BackendProject/WebAPIs/PREMIUMAGENCY/Custom.cs:22]: This is a better fit between #DEBUG brackets

File: BackendProject/WebAPIs/PREMIUMAGENCY/Custom.cs [hunk 3]
```diff
@@ -43,6 +53,11 @@ public class Custom
                         return File.ReadAllText($"{workpath}/eventController/MikuLiveEvent/localgetUserEventCustom.xml");
                     LoggerAccessor.LogInfo($"[PREMIUMAGENCY] - GetUserEventCustom sent for LOCAL MikuLiveEvent {eventId}!");
                     break;
+                case "81":
+                    if (File.Exists($"{workpath}/eventController/WhiteDay2010/getUserEventCustom.xml"))
+                        return File.ReadAllText($"{workpath}/eventController/WhiteDay2010/getUserEventCustom.xml");
+                    LoggerAccessor.LogInfo($"[PREMIUMAGENCY] - GetUserEventCustom sent for PUBLIC WhiteDay2010 {eventId}!");
+                    break;
```
Comment (GitHubProUser67) [inline BackendProject/WebAPIs/PREMIUMAGENCY/Custom.cs:60]: This is a better fit between #DEBUG brackets

File: BackendProject/WebAPIs/PREMIUMAGENCY/Event.cs [hunk 4]
```diff
@@ -27,6 +27,15 @@ public class Event
                          "<error_message type=\"text\">None</error_message>\r\n\r\n\t" +
                          "<status type=\"int\">0</status>\r\n" +
                          "</xml>";
+                case "81":
+                    LoggerAccessor.LogInfo($"[PREMIUMAGENCY] - CheckEvent sent for Public WhiteDay2010 {eventId}!");
+                    return "<xml>\r\n\t" +
```
Comment (GitHubProUser67) [inline BackendProject/WebAPIs/PREMIUMAGENCY/Event.cs:32]: This is a better fit between #DEBUG brackets

File: BackendProject/WebAPIs/PREMIUMAGENCY/Event.cs [hunk 5]
```diff
@@ -85,6 +94,15 @@ public class Event
                          "<error_message type=\"text\">None</error_message>\r\n" +
                          "<status type=\"int\">1</status>\r\n" +
                          "</xml>";
+                case "81":
+                    LoggerAccessor.LogInfo($"[PREMIUMAGENCY] - EntryEvent sent for PUBLIC WhiteDay2010 {eventId}!");
+                    return "<xml>\r\n\t" +
```
Comment (GitHubProUser67) [inline BackendProject/WebAPIs/PREMIUMAGENCY/Event.cs:99]: This is a better fit between #DEBUG brackets

File: BackendProject/WebAPIs/PREMIUMAGENCY/Custom.cs (hunk 2 above)
Comment (Jump-Suit) [reply BackendProject/WebAPIs/PREMIUMAGENCY/Custom.cs:22]: What do you mean this part is normal Info like it should...

**Review by GitHubProUser67 — APPROVED** [review]

**Pull request merged**